Source/GUI/dvrescue/dvrescue/AboutDialog.qml

```diff
@@ -0,0 +1,75 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import Qt.labs.settings 1.0
+import Launcher 0.1
+import FileUtils 1.0
+import SettingsUtils 1.0
+import QwtQuick2 1.0
+
+Popup {
+    id: aboutDialog
+    property string version: ""
+    property string buildDate: ""
+    property string buildQtVersion: ""
+    property string runtimeQtVersion: ""
+    property string buildFFMpegVersion: ""
+    property string runtimeFFMpegVersion: ""
+
+    anchors.centerIn: parent
+    contentWidth: layout.childrenRect.width
+    contentHeight: layout.childrenRect.height
+
+    ColumnLayout {
+        id: layout
+        Image {
+            source: "/dvrescue.png"
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        Text {
+            text: "DVRescue"
+            font.pixelSize: 25
+            Layout.alignment: Qt.AlignHCenter
+        }
+
+        RowLayout {
+            Layout.alignment: Qt.AlignHCenter
+            Text {
+                text: "Version"
+                font.pixelSize: 20
+            }
+
+            Text {
+                text: version
+                font.pixelSize: 20
+            }
+        }
+
+        Text {
+            text: buildDate
```

Comment:
can you move the buildDate to the end of the above line. Such as `Version $Version - $buildDate`